.storybook/main.js

```diff
@@ -17,9 +17,13 @@ module.exports = {
         enableShortcuts: false,
         isToolshown: true,
         storySort: undefined,
+        configureJSX: false,
     },
+}
+
+addParameters({
```

Comment:
@mavam - This makes sure the dark-mode works again (storybook upgrade needs a different config apparently)